src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/config-detail.jelly

```diff
@@ -2,27 +2,47 @@
 <j:jelly xmlns:j="jelly:core" xmlns:c="/lib/credentials"
          xmlns:f="/lib/form"
          xmlns:scm="/jenkins/scm/api/form">
-  <j:choose>
-    <j:when test="${descriptor.apiUriSelectable}">
-    <f:entry title="${%API endpoint}" field="apiUri">
-      <f:select/>
-    </f:entry>
-    </j:when>
-    <j:otherwise>
-      <f:invisibleEntry>
-        <f:select field="apiUri"/>
-      </f:invisibleEntry>
-    </j:otherwise>
-  </j:choose>
-  <f:entry title="${%Credentials}" field="credentialsId">
-    <c:select checkMethod="post"/>
-  </f:entry>
-  <f:entry title="${%Owner}" field="repoOwner">
-    <f:textbox/>
-  </f:entry>
-  <f:entry title="${%Repository}" field="repository">
-    <f:select/>
-  </f:entry>
+  <f:radioBlock name="url" value="Github Repository" checked="${instance.isURLRepo() == ''}" title="Github Repository Scan" inline="true">
+    <f:nested>
+      <j:choose>
+        <j:when test="${descriptor.apiUriSelectable}">
+          <f:entry title="${%API endpoint}" field="apiUri">
+            <f:select/>
+          </f:entry>
+        </j:when>
+        <j:otherwise>
+          <f:invisibleEntry>
+            <f:select field="apiUri"/>
+          </f:invisibleEntry>
+        </j:otherwise>
+      </j:choose>
+      <f:entry title="${%Credentials}" field="credentialsId">
+        <c:select checkMethod="post"/>
+      </f:entry>
+      <f:entry title ="${%Owner}" field="repoOwner">
+        <f:textbox/>
+      </f:entry>
+      <f:entry title="${%Repository}" field="repository">
+        <f:select/>
+      </f:entry>
+
+    </f:nested>
+  </f:radioBlock>
+  <f:radioBlock name="url" value="Github Repository" checked="${instance.isURLRepo() == 'true'}" title="Github Repository HTTPs URL" inline="true">
+    <f:nested>
+      <f:entry title="${%Credentials}" field="credentialsIdRaw">
```

Comment:
Why not just reuse `credentialsId`? Does it not work or something? It would be nice to not need a redundant `credentialsIdRaw` field.